connaisseur/flask_application.py

```diff
@@ -153,6 +153,17 @@ async def __validate_image(type_index, image, admission_request):
     type_, index = type_index
     logging_context.update(image=original_image)
 
+    # a resource that is being updated, without any image reference changes,
+    # can be automatically skipped, as image validation happened at some
+    # point prior
+    update_approval_on = os.environ.get("AUTOMATIC_UPDATE_APPROVAL", "0") == "1"
```

Comment:
Guess we didn't do it prior, but maybe it's time to namespace our env vars and make it `CONNAISSEUR_*`?

Reply:
will do in another commit, but will have to do that after you reviewed the chnages and I can do some rebasing and shuffeling of commits.

Reply:
will do in a different MR